proprietary/Components/src/denhaag/alert.tokens.json

```diff
@@ -1,126 +1,217 @@
 {
   "denhaag": {
     "alert": {
-      "border-radius": { "value": "1px" },
-      "padding-block-start": { "value": "{denhaag.space.block.xl}" },
-      "padding-block-end": { "value": "{denhaag.space.block.xl}" },
-      "padding-inline-start": { "value": "{denhaag.space.block.xl}" },
-      "padding-inline-end": { "value": "{denhaag.space.block.xl}" },
+      "border-radius": {
+        "value": "1px"
+      },
+      "padding-block-start": {
```

Comment:
Als de `start` en `end` altijd gelijk is zou ik zeggen combineer ze ;-) scheelt nogal wat css output